specification/terraform/Microsoft.AzureTerraform.Management/client.tsp

```diff
@@ -0,0 +1,8 @@
+import "@azure-tools/typespec-client-generator-core";
+import "./main.tsp";
+
+using Azure.ClientGenerator.Core;
+using Microsoft.AzureTerraform;
+
+@@clientName(ExportTerraform.exportTerraform, "ExportTerraform", "csharp");
+@@clientName(ExportTerraform.operationStatuses, "OperationStatuses", "csharp");
```

Comment:
usually we would like to hide LRO related operations in the dotnet SDK as well.
Could we add a `@@access(ExportTerraform.operationStatuses, internal)` to it?

Reply:
Added in e669bc3b30, thx.